common/unsafe/src/test/java/org/apache/spark/unsafe/types/UTF8StringSuite.java

```diff
@@ -465,27 +465,27 @@ public void translate() {
     assertEquals(
       fromString("1a2s3ae"),
       fromString("translate").translate(ImmutableMap.of(
-        'r', '1',
-        'n', '2',
-        'l', '3',
-        't', '\0'
+        "r", "1",
```

Comment:
Why do these need to be strings now? all seem to be 1 char.

Reply:
Ah, it's a wrong change.

Reply:
The type parameter of `translate` is changed from `Map<Character, Character>` to `Map<String, String>` to support >=`U+10000` unicode characters so this change is necessary. Those characters can't be represent as `Character`.